modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -1367,10 +1367,6 @@ func (suite *KeeperTestSuite) TestWriteUpgradeCancelChannel() {
 				}
 
 				ibctesting.AssertEvents(&suite.Suite, expEvents, events)
-
```

Comment:
ah sorry, my comment might of been somewhat confusing. `CounterpartyUpgrade` being deleted should still be checked, we should just move the check further up, right after we check that the chain upgrade has been deleted:

I.e add the removed code right after line 1348 that checks that the `Upgrade` has been deleted.